test/scripts/crc_16.py

```diff
@@ -0,0 +1,74 @@
+_low_crc = [
```

Comment:
This could be done by modifying the COSMOS compose file. Binding only the script directory and any needed utilities avoids issues with the COSMOS environment and symlinks in the broader Adamant environment. Something like this would work:
```
openc3-cosmos-script-runner-api:
  volumes:
    - "./../adamant_example/test/scripts:/plugins/DEFAULT/targets_modified:ro"
    - "./../adamant/gnd/util:/plugins/DEFAULT/targets_modified:ro"
```

Reply:
Install scripts now copy this file directly from Adamant, rather than from `test/scripts`, in [84d2ca9](https://github.com/lasp/adamant_example/pull/33/commits/84d2ca97a379b45b6513649529bdee7312d4c31e)